src/neo-vm/OpCode.cs

```diff
@@ -310,74 +310,270 @@ public enum OpCode : byte
 
         #endregion
 
-        #region Old opcodes
+        #region Slot
 
         /// <summary>
-        /// Puts the input onto the top of the alt stack. Removes it from the main stack.
+        /// Initialize the static field list for the current execution context.
         /// </summary>
-        TOALTSTACK = 0x6B,
+        [OperandSize(Size = 1)]
+        INITSSLOT = 0x56,
         /// <summary>
-        /// Puts the input onto the top of the main stack. Removes it from the alt stack.
+        /// Initialize the argument slot and the local variable list for the current execution context.
         /// </summary>
-        FROMALTSTACK = 0x6C,
```

Comment:
Do we have any operation that takes value from Slot and "removes" it? I mean, takes from last index of slot and reduces its size? It could be similar to the FROMALTSTACK... but I guess on practice we won't need it, compilers should be able to generate slot logic directly, instead of double stacks.

Reply:
No. Slot size can't be changed.